Controllers/RankController.cs

```diff
@@ -1182,6 +1186,8 @@ public ActionResult<PrevQualification> PrevQualificationTime(string hash)
                 }
 
             }
+            // TODO: Do we need to add something here for calculating clan ranking?
```

Comment:
This endpoint is obsolete and can be deleted

Reply:
But we will need to make the admin endpoint to recalculate clan ranking for all ranked leaderboards.

Reply:
Fixed